package/src/components/PaymentsCheckoutAction/v1/PaymentsCheckoutAction.js

```diff
@@ -0,0 +1,297 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import { withComponents } from "@reactioncommerce/components-context";
+import styled from "styled-components";
+import { addTypographyStyles, CustomPropTypes, formatMoney } from "../../../utils";
+
+const Title = styled.h3`
+  ${addTypographyStyles("PaymentsCheckoutActionTitle", "subheadingTextBold")}
+`;
+
+const ActionCompleteDiv = styled.div`
+  ${addTypographyStyles("PaymentsCheckoutActionComplete", "bodyText")};
+`;
+
+class PaymentsCheckoutAction extends Component {
+  static renderComplete({ payments }) {
+    if (!Array.isArray(payments) || payments.length === 0) return null;
+
+    const paymentLines = payments.map(({ displayName, payment }, index) => (
+      <div key={`${index}`}>{displayName}{payment.amount ? ` (${formatMoney(payment.amount)})` : null}</div>
+    ));
+
+    return (
+      <ActionCompleteDiv>{paymentLines}</ActionCompleteDiv>
+    );
+  }
+
+  static propTypes = {
+    /**
+     * Provide the shipping address and any other known addresses.
```

Comment:
Possible copy update, same as mentioned before, maybe call the addresses "saved" addresses instead of known.